frontend/src/components/FormEnter/FormEnter.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+
+import Input from '../Input/Input';
+import Button from '../Button/Button';
+
+import '../../../../styles/FormEnter.css';
+
+function FormEnter() {
+	const [activeRegistration, setActiveRegistration] = useState(true);
+	const [activeEnter, setActiveEnter] = useState(false);
+	const [activeRecovery, setActiveRecovery] = useState(false);
+	const [activeTitle, setActiveTitle] = useState(false);
+
+	const handleClickEnter = () => {
+		setActiveRegistration(false);
+		setActiveEnter(true);
+		setActiveRecovery(false);
+		setActiveTitle(false);
+	}
+
+	const handleClickRegistration = () => {
+		setActiveRegistration(true);
+		setActiveEnter(false);
+		setActiveRecovery(false);
+		setActiveTitle(false);
+	}
+
+	const handleClickRecovery = () => {
+		setActiveRegistration(false);
+		setActiveEnter(false);
+		setActiveRecovery(true);
+		setActiveTitle(true);
+	}
```

Comment:
Предлагаю стейт тоже модифицировать, на мой взгляд сейчас кода много и это выглядит перегружено. По сути компонент формы у нас может принимать 3 состояния: registration, entry и recovery. Можно за пределами компонента создать объект с этими значениями и кода станет гораздо меньше:
```suggestion
const formState = {
	registration: 'registration',
	entry: 'entry',
	recovery: 'recovery',
}

function FormEnter() {
	const [formView, setFormView] = useState(formState.registration)

	const handleClickEnter = () => {
		setFormView(formState.entry)
	}

	const handleClickRegistration = () => {
		setFormView(formState.registration)
	}

	const handleClickRecovery = () => {
		setFormView(formState.recovery)
	}

```